app/models/request.rb

```diff
@@ -114,4 +114,12 @@ def generate_puppet_name_script
   def url(host_name)
     Rails.application.routes.url_helpers.request_url self, host: host_name
   end
+
+  def gibi_to_mibi(gibi)
+    gibi * 1024
+  end
+
+  def gibi_to_kibi(gibi)
+    gibi * 1024**2
```

Comment:
please use 2**(10+2)

Reply:
or (2**10) **2

Reply:
or 2**20

Reply:
or 2**(10*2)

Reply:
or 2**(2**3+2) **2

Reply:
or 2**(2**(2+2/2)+2) **2

Reply:
or make a method "binary_base", returning 2 and change formula to
binary_base**(binary_base**(binary_base + binary_base/binary_base)+ binary_base) ** binary_base

Reply:
> or make a method "binary_base", returning 2 and change formula to
> binary_base**(binary_base**(binary_base + binary_base/binary_base)+ binary_base) ** binary_base

that way we have no magic numbers

Reply:
@BjoernDaase we should remove all magic numbers that way

Reply:
@Kenneth-Schroeder Are you sober?